src/com/projectkorra/rpg/worldevent/IWorldEvent.java

```diff
@@ -34,6 +34,13 @@
 	 * @return list of attributes
 	 */
 	public List<String> getAttributes();
+	
+	/**
+	 * A list of worldevents which cannot concurrently occur with this worldevent
+	 * 
+	 * @return blacklisted events
+	 */
+	public List<String> getBlacklistedEvents();
```

Comment:
Maybe we should have this be a `List<WorldEvent>` with a parser on construction so that we don't have to check for exact name matching a bunch.

Reply:
The problem is what if an event in the list isn't loaded yet